### ISSUE TITLE
No sound on either end

### ISSUE THREAD

**Issue by TheDataLeek** [Mariatta/enhanced-coc-hotline#53]
Good evening, I'm part of PyColorado and I'm having trouble getting the hotline set up. We've followed the instructions, and everything is almost working, except that after calls have been placed to individuals on the list of recipients, neither end can hear anything past the initial Talk action. There's no music, and I don't think the call itself actually gets connected.

Any ideas as to what we're doing wrong?

Thanks!
Zoe

**Comment by Mariatta** [comment]
Hi, thanks for trying out this hotline.

To make sure I understand correctly, you were able to dial the hotline, heard the "Welcome to the PyColorado hotline", but then you don't hear the music, and it doesn't get forwarded to anyone?

Of you did not hear the "Welcome to ... " message in the first place?

**Comment by TheDataLeek** [comment]
Yep, dialing in works, we hear the intro, but no music, and then line goes quiet. Meanwhile, on the other end, the call is forwarded, the CoC responders can answer, the blurb about "you're about to be connected" is played, and then the line goes silent as well. The call will then keep calling more people instead of stopping on the first person that picks up.
